pylint_odoo/checkers/no_modules.py

```diff
@@ -327,10 +348,29 @@ class NoModuleChecker(BaseChecker):
         }),
     )
 
+    def open(self):
+        super(NoModuleChecker, self).open()
+        self.config.deprecated_field_parameters = \
+            self.colon_list_to_dict(self.config.deprecated_field_parameters)
+
+    def colon_list_to_dict(self, colon_list=None):
+        """ \
+        Converts a colon list to a dictionary
+        Example:
+            >> self.colon_list_to_dist(['colon:list', 'just_key'])
+            {'colon': 'list', 'just_key': None}
+        """
+        tmp_dict = {}
+        for item in colon_list if isinstance(colon_list, list) else []:
+            prop = item.split(":")
+            tmp_dict[prop[0]] = prop[1] if len(prop) > 1 else None
```

Comment:
We don't need a extra validation here since that we are using a good help from `option` definition